R/dvhydrograph-data.R

```diff
@@ -126,17 +138,28 @@ parseDVMinMaxIVs <- function(reportObject, timezone, type, invertedFlag, exclude
 #' vertical lines connecting the steps between those TS.
 #' @param stat the parsed non-estimated time series
 #' @param est the parsed estimated time series
+#' @param excludeZeroNegativeFlag whether or not zero / negative values
+#' will be removed
 #' @return a list of vertical lines connecting steps between stat and est
 #' @importFrom dplyr arrange
-getEstimatedEdges <- function(stat, est){
+#' @return a list of the edges with the folowing columns time, y0, y1,
```

Comment:
it looks like you have two `@return` statements - I think you'll need to delete one.